src/output.rs

```diff
@@ -34,12 +36,16 @@ pub fn handle_hook_error(err: Error, shellpid: u32, silent: bool) -> i32 {
     return 1;
 }
 
-pub fn print_activation(activated: bool) {
-    let word = match activated {
-        true => "activated",
-        false => "deactivated",
-    };
-    eprintln!("\x1b[1;34m{} {}.\x1b[0m", word, SHADOWENV);
+pub fn print_activation(activated: bool, features: HashSet<Feature>) {
+    if activated {
+        eprint!("\x1b[1;34mactivated {}\x1b[1;34m", SHADOWENV);
+        for feature in features {
+            eprint!(" ({})", feature);
```

Comment:
I think I'd prefer it to look like:

```
activated shadowenv (ruby:x.y.z, node:a.b.c)
```

Thoughts?

Reply:
```rust
        features
            .iter()
            .map(|s| format!("{}", s))
            .collect::<Vec<String>>()
            .join(", ");
```

Reply:
Yeah, that's nicer.